### ISSUE TITLE
alerts break AOT 

### ISSUE THREAD

**Issue by DexterHaslem** [valor-software/ngx-bootstrap#2266]
just upgraded from old ng2-bootstrap to ngx-bootstrap 1.8.0

following the getting started for angular-cli (https://github.com/valor-software/ngx-bootstrap/blob/development/docs/getting-started/ng-cli.md), added AlertModule.forRoot() to app module imports.

Once that is done, `ng build -prod` no longer works:
```
ERROR in ./src/main.ts
Module not found: Error: Can't resolve './$$_gendir/app/app.module.ngfactory' in 'C:\goprojects\src\fatc\angular_client\src'
 @ ./src/main.ts 6:0-74
 @ multi ./src/main.ts
```
removing alerts module, AOT works again. `ng build -prod --aot false` does work. AOT did work on older ng2-bootstrap w/ alerts enabled if that helps

**Comment by mark-siama** [comment]
I can confirm my project (private repo) is getting the same build errors with AoT. We managed to reproduce it by building a new Angular-CLI app and adding the `<template></template>` tag which is now deprecated for `ng-template`

**Comment by IlyaSurmay** [comment]
which version of angular CLI have you used? there was a problem with AOT in 1.2.3

**Comment by valorkin** [comment]
known issue of ng cli v1.2.*
fixed in @angular/cli v1.2.4
